src/libraries/System.Runtime.InteropServices/tests/System.Runtime.InteropServices.UnitTests/System/Runtime/InteropServices/ComVisibleAttributeTests.cs

```diff
@@ -8,15 +8,15 @@ namespace System.Runtime.InteropServices.Tests
     [ComVisible(true)]
     public class ComVisibleAttributeTests
     {
-        [Fact]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsBuiltInComEnabled))]
```

Comment:
Wouldn't this work even without COM? (Same below.)

I think there will be a NativeAOT failure, but that's https://github.com/dotnet/runtimelab/issues/830.